Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -1538,4 +1539,41 @@ theorem multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multipl
     multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity hr ha]
 #align multiplicity_normalized_factors_equiv_span_normalized_factors_symm_eq_multiplicity multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multiplicity
 
+/-- The bijection `normalizedFactorsEquivSpanNormalizedFactors.symm` between the set of prime
+    factors of the ideal `⟨r⟩` and the set of prime factors of `r` preserves `count` of the
+    corresponding multisets. See
+    `multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity` for the version
+    stated in terms of multiplicity. -/
+theorem count_normalizedFactorsSpan_eq_count {r X : R} (hr : r ≠ 0) (hX₁ : normUnit X = 1)
+    (hX : Prime X) : Multiset.count X (normalizedFactors r) =
+      Multiset.count (Ideal.span {X} : Ideal R) (normalizedFactors (Ideal.span {r})) := by
+    have := multiplicity_eq_multiplicity_span (R := R) (a := X) (b := r)
+    rw [multiplicity_eq_count_normalizedFactors
+    (Prime.irreducible hX) hr,
+        multiplicity_eq_count_normalizedFactors] at this
+    · simp_all only [ne_eq, normalize_apply, normUnit_eq_one, Units.val_one, one_eq_top, mul_top,
+      mul_one, Nat.cast_inj]
+    · apply Prime.irreducible
+      dsimp only [Set.mem_setOf_eq, Set.coe_setOf, normalizedFactorsEquivSpanNormalizedFactors,
+        Equiv.ofBijective_apply]
+      simpa only [prime_span_singleton_iff]
+    · simp only [Submodule.zero_eq_bot, ne_eq, span_singleton_eq_bot, hr, not_false_eq_true]
```

Comment:
The proof should be indented two spaces to start with, not four

Reply:
It is indented with two spaces, no?